src/azure-cli/HISTORY.rst

```diff
@@ -117,6 +117,8 @@ Release History
 
 **Redis**
 
+* Add preview param `--replicas-per-master` to `az redis create` command
```

Comment:
I think you add the new param for both `create`   and `update` commands, right?

Reply:
ReplicasPerMaster is under preview and want to support for `create` command only. `update` operation still needs some work/improvements before it can be exposed through CLI.